### PR TITLE
feat: simple table

### LINKED ISSUE #1726: SimpleTable: import SimpleTable from giraffe
### Background
Depends on https://github.com/influxdata/giraffe/issues/618

This is the UI component of the above Giraffe ticket. After `SimpleTable` exists in giraffe, we should make Notebooks use the giraffe version of `SimpleTable`

We might want to consider feature flagging this, depending on the state of how it's used.

### Acceptance Criteria
- Notebooks (and any other consumers of `SimpleTable`) use Giraffe's `SimpleTable`

### PR DESCRIPTION
**Pull request by ChitlangeSahas**
Closes #1726

Brings in Simple table from `giraffe`.  The changes are behind the `useGiraffeGraphs` feature flag.

https://user-images.githubusercontent.com/18511823/130877218-a1274ca2-e165-4f85-8174-73534f1f6d24.mov

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 59 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2415)

File: src/visualization/types/SimpleTable/properties.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import {SimpleTableViewProperties} from 'src/types'
+import {defaultViewQuery} from 'src/views/helpers/index'
```
[the remark this answers is not in the document]
Comment (drdelambre) [reply src/visualization/types/SimpleTable/properties.ts:2]: see next comment

File: src/visualization/types/SimpleTable/view.tsx [hunk 2]
```diff
@@ -17,14 +18,18 @@ interface Props extends VisualizationProps {
...
-    </div>
-  )
+  const parsed = parseFromFluxResults(result)
+  const fluxResponse = parsed.tableData.join('\n')
+  const config: Config = {
+    fluxResponse,
+    layers: [
+      {
+        type: 'simple table',
+        showAll: properties.showAll,
```
[the remark this answers is not in the document]
Comment (drdelambre) [reply src/visualization/types/SimpleTable/view.tsx:28]: and it's thrown away because this isn't used in giraffe

File: src/visualization/types/SimpleTable/index.tsx [hunk 3]
```diff
@@ -33,6 +33,7 @@ export default register => {
     graphic: icon,
     initial: {type: 'simple-table', showAll: false},
     component: view,
+    featureFlag: 'useGiraffeGraphs',
```
Comment (drdelambre) [inline src/visualization/types/SimpleTable/index.tsx:36]: what is your understanding of this property?

Comment (ChitlangeSahas) [reply src/visualization/types/SimpleTable/index.tsx:36]: It seems like a convenient way to put the featureFlag in one place and then use it for that graph, i noticed the UI access those properties when deciding what graph types to expose in the UI, through the dropdown.

File: src/visualization/types/SimpleTable/view.tsx [hunk 4]
```diff
@@ -17,14 +18,18 @@ interface Props extends VisualizationProps {
...
 const SimpleTable: FC<Props> = ({properties, result}) => {
-  return (
-    <div className="visualization--simple-table" data-testid="simple-table">
-      <PaginationProvider total={result?.table?.length || 0}>
-        <PagedTable properties={properties} result={result} />
-        <PageControl />
-      </PaginationProvider>
-    </div>
-  )
+  const parsed = parseFromFluxResults(result)
```
Comment (drdelambre) [inline src/visualization/types/SimpleTable/view.tsx:21]: you need to revert these changes, and wrap them behind a feature flag. the way you have the flag set up currently is that you're preventing the user from using simpletable, but still making all existing visualizations of this type fo through the new logic. we want to gate the code change, not the user experience for this visualization.

Comment (ChitlangeSahas) [reply src/visualization/types/SimpleTable/view.tsx:21]: Good point!! Thank you for noticing that, that was a blunder on my part, I get it now. I'll fix that asap.

Comment (drdelambre) [reply src/visualization/types/SimpleTable/view.tsx:26]: this declaration also needs to be put inside of the flag or else we're double parsing (an expensive operation) the results for every simpletable

Comment (ChitlangeSahas) [reply src/visualization/types/SimpleTable/view.tsx:26]: Ah! Got it, will fix that 👍

Comment (ChitlangeSahas) [reply src/visualization/types/SimpleTable/view.tsx]: Just fixed it.

**Review by drdelambre — APPROVED** [review]
